testsuite/features/secondary/min_salt_mgrcompat_state.feature

```diff
@@ -106,3 +106,19 @@ Feature: Verify that Salt mgrcompat state works when the new module.run syntax i
     And I click on "Bootstrap"
     And I wait until I see "Bootstrap process initiated." text
     And I wait until onboarding is completed for "sle_minion"
+
+  Scenario: Cleanup: restore channels on the minion after mgrcompat tests
```

Comment:
Thoses changes don't seem related to you PR.

Reply:
Indeed, thank you